Description: Skip ansi::tests::contrast on x86
 Debian's x86 baseline differs from upstream Rust, which causes this to fail due to the differing float support.
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: vte/src/ansi.rs
===================================================================
--- vte.orig/src/ansi.rs
+++ vte/src/ansi.rs
@@ -2087,7 +2087,7 @@ mod tests {
     }
 
     #[test]
-    #[cfg(not(feature = "no_std"))]
+    #[cfg(all(not(feature = "no_std"), not(target = "x86")))]
     fn contrast() {
         let rgb1 = Rgb { r: 0xff, g: 0xff, b: 0xff };
         let rgb2 = Rgb { r: 0x00, g: 0x00, b: 0x00 };
